controllers/backup.go

```diff
@@ -127,7 +127,7 @@ func (r *ClusterGroupUpgradeReconciler) backupPreparing(ctx context.Context, clu
 		return currentState, err
 	}
 
-	spec := r.getBackupJobTemplateData(clusterGroupUpgrade, cluster)
+	spec, err := r.getBackupJobTemplateData(clusterGroupUpgrade, cluster)
```

Comment:
This is interesting, the code below wasn't checking this error at all as it wasn't being returned.

Reply:
overlooked previously